library/database/class.database.php

```diff
@@ -153,9 +153,8 @@ protected function newPDO($Dsn, $User, $Password) {
             $PDO = new PDO(strtolower($this->Engine).':'.$Dsn, $User, $Password, $this->ConnectionOptions);
             $PDO->setAttribute(PDO::ATTR_EMULATE_PREPARES, 0);
 
-            if ($this->ConnectionOptions[1002]) {
-                $PDO->query($this->ConnectionOptions[1002]);
-            }
+            $encoding = c('Database.CharacterEncoding', 'utf8mb4');
```

Comment:
I'd say this is a bit too soon and looks sloppy without everything else going to utf8mb4.